src/validation.cpp

```diff
@@ -310,6 +310,9 @@ bool CheckSequenceLocks(const CTransaction &tx, int flags, LockPoints* lp, bool
 }
 
 
+// Returns the script flags which should be checked for a given block
```

Comment:
It feels much more reasonable next to the consensus code, as it is really critical consensus logic, instead of mixed in with all the ATMP code.